test/integration/testdata/packagecollision/interface.go

```diff
@@ -0,0 +1,10 @@
+package packagecollision
+
+import (
+	afoo "github.com/Makpoc/gomigen/test/integration/testdata/packagecollision/a/foo"
+	bfoo "github.com/Makpoc/gomigen/test/integration/testdata/packagecollision/b/foo"
+)
+
+type Compare interface {
+	Equals(afoo.Foo, bfoo.Foo) bool
```

Comment:
If you really want to test package collision, you should include two more scenarios (can be added as additional arguments to this method) that in my personal experience break tools (and are hard to get right).

1. Having and using a dot import
2. Having a package that ends with non-`foo` suffix (e.g. `github.com/Makpoc/gomigen/test/integration/testdata/packagecollision/a/bar`)  but internally declares itself as `package foo`.

Reply:
Added tests for both dot import and custom package name